docs/commands/job/run.md

```diff
@@ -28,10 +30,12 @@ This command runs a job in Studio using the specified query file. You can config
 * `--workers WORKERS` - Number of workers for the job
 * `--files FILES` - Additional files to include in the job
 * `--python-version PYTHON_VERSION` - Python version for the job (e.g., 3.9, 3.10, 3.11)
+* `--repository REPOSITORY` - Repository URL to clone before running the job
 * `--req-file REQ_FILE` - Python requirements file
 * `--req REQ` - Python package requirements
 * `--priority PRIORITY` - Priority for the job in range 0-5. Lower value is higher priority (default: 5)
-* `--repository URL` - Repository URL to clone before running the job.
+* `--start-time START_TIME` - Start time for the cron task. Supports various formats including natural language: '2024-01-15 14:30:00', 'tomorrow 3pm', 'next monday 9am', '2024-01-15T14:30:00Z', 'in 2 hours', 'Jan 15, 2024 2:30 PM', etc. If cron expression is provided, the cron job will activate after this time. Otherwise, the job will run once at this time.
```

Comment:
so, does it work only if cron also provided? can it be a single (delayed run?)

Reply:
Updated the docs.